fortran/test/fetch_content_integration/test_micm_api.F90

```diff
@@ -32,11 +32,12 @@ program test_micm_api
     real(real64) :: E_ = 0.0
   end type ArrheniusReaction
 
-  call test_api()
-  call test_multiple_grid_cell_vector_Rosenbrock()
-  call test_multiple_grid_cell_standard_Rosenbrock()
-  call test_multiple_grid_cell_vector_BackwardEuler()
-  call test_multiple_grid_cell_standard_BackwardEuler()
+  ! call test_api()
+  ! call test_multiple_grid_cell_vector_Rosenbrock()
+  ! call test_multiple_grid_cell_standard_Rosenbrock()
+  ! call test_multiple_grid_cell_vector_BackwardEuler()
+  ! call test_multiple_grid_cell_standard_BackwardEuler()
```

Comment:
should these be commented out?

Reply:
oh no! I forgot I had commented them out